consensus/consensus-types/src/block_test_utils.rs

```diff
@@ -191,14 +191,10 @@ pub fn gen_test_certificate(
     parent_block: BlockInfo,
     committed_block: Option<BlockInfo>,
 ) -> QuorumCert {
-    let vote_data = VoteData::new(block, parent_block);
+    let vote_data = VoteData::new(block, parent_block.clone());
     let ledger_info = match committed_block {
         Some(info) => LedgerInfo::new(info, vote_data.hash()),
-        None => {
-            let mut placeholder = placeholder_ledger_info();
-            placeholder.set_consensus_data_hash(vote_data.hash());
-            placeholder
-        },
+        None => LedgerInfo::new(parent_block, vote_data.hash()),
```

Comment:
this is wrong, why do we need this change? when commit block is None, the ledger info should carry the dummy value

Reply:
Actually, I just used `None` here as a shortcut so that we automatically commit the parent block instead of having to specify the value of the parent block again. I will change it as it is leading to a confusion.